pypeit/core/save.py

```diff
@@ -216,6 +217,82 @@ def save_1d_spectra_fits(specObjs, header, spectrograph, outfile, helio_dict=Non
     return outfile
 
 
+def save_coadd1d_to_fits(outfile, waves, fluxes, ivars, masks, telluric=None, obj_model=None,
```

Comment:
Same as above for `load`.  Almost all of these methods will disappear
soon, i.e. in v0.11.2